src/main/scala/com/ppb/rabbitconsumer/RabbitConsumer.scala

```diff
@@ -11,6 +12,9 @@ import scalaz.concurrent.Task
 import scalaz.stream._
 
 case class Cxn(filename: String, nextMessage: () => RabbitResponse, disconnect: () => Try[Unit])
+case class Sxn(subscription:((Try[RabbitResponse] => Unit) => String), disconnect: () => Try[Unit])
```

Comment:
not sure what exactly Cxn and Sxn mean (Connection and Session?)
can we use more explicit names to avoid confusion?